kornia/geometry/depth.py

```diff
@@ -110,6 +111,7 @@ def depth_to_3d_v2(
 # NOTE: this function should replace the old depth_to_3d
 
 
+@deprecated(replace_with="depth_to_3d_v2", version='0.7.0')
 def depth_to_3d(depth: Tensor, camera_matrix: Tensor, normalize_points: bool = False) -> Tensor:
```

Comment:
@johnnv1 i'm not sure if this deprecated decorator (or the message) is the right one. Ideally we want to preserve the same function signature but update the implementation of "v1" with "v2".

maybe something like this:
```python
@deprecated(extra_reason="the implementation with be replaced with depth_to_3d_v2", version='0.7.0')
def depth_to_3d(...)
```
and

```python
@deprecated(replace_with="use depth_to_3d", version='0.7.0')
def depth_to_3d_v2(...)
```

Reply:
can be it, it's kinda weird to have deprecated messages in the two functions -- but I think it should works

Reply:
another more practical way is to just simply rename `depth_to_3d_v2` -> `depth_to_3d` which will make backward compatible the existing functionality in case the grid is not provided

Reply:
well, if it's backward compatible, +1 from my side to just replace it